packages/plasma-ui/src/components/Button/ActionButton.tsx

```diff
@@ -22,11 +22,13 @@ import { applyInteraction, InteractionProps } from '../../mixins';
 
 import { sizes } from './ActionButton.sizes';
 
-export type ActionButtonProps = Omit<BaseProps, 'stretch' | 'pin'> &
+type ActionButtonBaseProps = Omit<BaseProps, 'stretch' | 'pin'> &
     Partial<ButtonSizeProps> &
     Partial<ButtonViewProps> &
     InteractionProps & { pin?: Extract<PinProps['pin'], 'square-square' | 'circle-circle'> };
 
+export type ActionButtonProps = Omit<ActionButtonBaseProps, 'nonce' | 'onResize' | 'onResizeCapture'>;
```

Comment:
давай тут такой же текст добавим, как в других местах:

```
// INFO: Omit 'onResize' | 'onResizeCapture' | 'nonce'
// because this types coming with @types/react@18.0.25 and breaks react@17.0.2 with @types/react@18.0.18
```